tmt/steps/provision/vagrant.py

```diff
@@ -392,18 +548,18 @@ def add_defaults(self):
              - disable nfs check
             and validates Vagrantfile
         """
-        self.add_synced_folder(".", "/vagrant", 'disabled: true')
+        self.add_synced_folder(".", "/vagrant", 'rsync', 'disabled: true')
 
-        dir = self.step.plan.workdir
-        self.add_synced_folder(dir, dir)
+        dir = self.parent.plan.workdir
```

Comment:
`dir` is a reserved word in python, better use something else.

Reply:
Right, thanks.